### ISSUE TITLE
User Guide: remove docker-compose reference

### ISSUE THREAD

**Issue by bertsky** [OCR-D/ocrd-website#317]
I see no point in linking to the installation instructions for docker-compose at this point. It is highly technical and we don't even use docker-compose (yet).

Also, the whole section [Setup docker](https://ocr-d.de/en/user_guide#setup-docker) really belongs to the Setup Guide. (Yes, there should be a back-reference to the setup guide, but the gist of the information should be there, not here.)

**Comment by bertsky** [comment]
ping @lena-hinrichsen

**Comment by lena-hinrichsen** [comment]
> Also, the whole section [Setup docker](https://ocr-d.de/en/user_guide#setup-docker) really belongs to the Setup Guide. (Yes, there should be a back-reference to the setup guide, but the gist of the information should be there, not here.)

Question about this: The step 

```
# Start docker daemon at startup
sudo systemctl enable docker
# Add user to group 'docker'
sudo usermod -aG docker $USER
```

only has to be done once (after the setup with Docker), am I correct? That's why it should be in the Setup Guide so it's redundant in the User Guide?

And:
Would you recommend to run `docker images` as a test once after installation or every time after starting the OS/shell/whatever?

**Comment by bertsky** [comment]
> Question about this: The step [...] only has to be done once (after the setup with Docker), am I correct? 

Yes, only once – sometimes it even comes with the system package for Docker as a post-install step.

> That's why it should be in the Setup Guide so it's redundant in the User Guide?

Yes. However, it does make sense to discern the two scenarios Docker vs native – they do have diverging prerequisites. For Docker, it's simply following the Docker part of the Setup Guide and perhaps logging into the Docker container, for native, it's activating your (install-time) venv in the current shell.

> 
> And: Would you recommend to run `docker images` as a test once after installation or every time after starting the OS/shell/whatever?

No – keep it simple.
